### DISCUSSION TITLE
allow ${home} in connection SSH private key field (e.g. "${home}\.ssh\id_rsa") so each user doesn't have to manually edit

### DISCUSSION THREAD

**Discussion by nutmix** [dbeaver/dbeaver#23400] [Ideas]
Currently, when you export your connections so that the rest of the development team can import them, each developer has to manually edit each connection to change "C:\Users\usera\.ssh\id_rsa" "C:\Users\userb\.ssh\id_rsa" which is extremely tedious.
If we could use ${home} in the private key location, the connections would work out of the box for all users.
This change request would allow variable substitution in the ssh tunnel private key file location field.

**Comment by LonwoLonwo** [comment]
Hello @nutmix

However, as far as I can see, we already support this parameter in the SSH connection field. Did you try to use it?

![2024-04-23 20_20_37-Connection _dvdrental 2 1_ configuration](https://github.com/dbeaver/dbeaver/assets/45152336/ec731bd2-b1b4-4fc5-a104-978573bec5e0)

**Reply by nutmix** [reply]
Yes, I tried that, it doesnt work.  Looks like those variables are not supported in private key path, which is what I am hoping would get implemented.

**Reply by LonwoLonwo** [reply]
How exactly doesn't it work, and how are you using this parameter?
